projects/storefrontlib/src/cms-components/configurator/generic/service/config-router-extractor.service.ts

```diff
@@ -14,9 +14,11 @@ import { ConfigurationRouter } from './config-router-data';
  */
 @Injectable({ providedIn: 'root' })
 export class ConfigRouterExtractorService {
+  protected routeFragmentConfigure = 'configure';
+  protected routeFragmentConfigureOverview = 'configureOverview';
```

Comment:
Bagatelle: what about making them `readonly` (and uppercase)? I believe we can drop `fragment` since we operate on sematic routes (not on URL fragments).

Reply:
readonly and uppercase: Yes, will do in another PR.
I guess we should keep fragment, as the routes are configureTEXTFIELD, configureOverviewCPQCONFIGURATOR